packages/squiggle-lang/src/rescript/Utility/Errors.res

```diff
@@ -0,0 +1,31 @@
+type property = {fnName: string, propertyName: string}
```

Comment:
Is the goal for this file to collect error types all over the codebase all the time?